docs/install-cpu-source.rst

```diff
@@ -5,7 +5,7 @@
 
     # CPU version of pytorch has smaller footprint - see installation instructions in
     # pytorch documentation - https://pytorch.org/get-started/locally/
-    pip3 install "torch>=1.0,<1.12+cpu" -f https://download.pytorch.org/whl/cpu/torch_stable.html
+    pip3 install "torch>=1.0,<1.13+cpu" -f https://download.pytorch.org/whl/cpu/torch_stable.html
```

Comment:
@Innixma @yinweisu I revised our installation guide so that users can install torch 1.12

Reply:
Are we still claiming support for torch 1.0? Should we tighten bounds?

Reply:
How about just recommending the latest torch in our tutorial? I think we should work for torch>=1.9.

Reply:
I checked the requirement from torchmetrics, it is currently asking for torch 1.11 in the stable version. Thus, we may also recommend using torch 1.11 in our website for now and upgrade to torch 1.12 once that is stable. Meanwhile, I think we can turn on torch 1.12 in our unittests.